modules/eks/actions-runner-controller/README.md

```diff
@@ -37,6 +37,21 @@ components:
         github_app_id: "REPLACE_ME_GH_APP_ID"
         github_app_installation_id: "REPLACE_ME_GH_INSTALLATION_ID"
 
+        # use to enable docker config json secret, which can login to dockerhub for your GHA Runners
+        docker_config_json_enabled: true
+        # The content of this param should look like:
+        # {
+        #  "auths": {
+        #    "https://index.docker.io/v1/": {
+        #      "username": "your_username",
+        #      "password": "your_password
+        #      "email": "your_email",
+        #      "auth": "$(echo "your_username:your_password" | base64)"
+        #    }
+        #  }
+        # } | base64
+        ssm_docker_config_json_path: "/github_runners/docker/config-json"
```

Comment:
Document how to set this value in SSM

Reply:
Ideally with an atmos workflow

Reply:
@osterman here's the Atmos workflow I created for setting the value in SSM

```shell
  save/docker-config-json:
    description: Prompt for uploading Docker Config JSON to the AWS SSM Parameter Store
    steps:
      - type: shell
        command: |-
          echo "Please enter the Docker Config JSON file path"
          echo "See https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry for information on how to create the file"
          read -p "Docker Config JSON file path: " -r DOCKER_CONFIG_JSON_FILE_PATH
          if [ -z "DOCKER_CONFIG_JSON_FILE_PATH" ]
          then
              echo 'Inputs cannot be blank please try again!'
              exit 0
          fi

          DOCKER_CONFIG_JSON=$(<$DOCKER_CONFIG_JSON_FILE_PATH);
          ENCODED_DOCKER_CONFIG_JSON=$(echo "$DOCKER_CONFIG_JSON" | base64 -w 0 );

          echo $DOCKER_CONFIG_JSON
          echo $ENCODED_DOCKER_CONFIG_JSON

          AWS_PROFILE=cch-core-gbl-auto-admin

          set -e

          chamber write github_runners/docker config-json -- "$ENCODED_DOCKER_CONFIG_JSON"

          echo 'Saved Docker Config JSON to the AWS SSM Parameter Store'
```